backend/server/sql/migrations/20250426_drv_eval.sql

```diff
@@ -0,0 +1,26 @@
+-- This is the minimal amount of information needed to identify a build
+-- This purposely tries to avoid details such as attr path which may
+-- differ (e.g. python3.pkgs.setuptools vs python3Packages.setuptools)
+CREATE TABLE IF NOT EXISTS Drv (
+    id INTEGER PRIMARY KEY,
```

Comment:
Now thinking about this more, I would just like to get rid of this to avoid having to context switch between "what the DB thinks" and what "nix thinks"